lib/Utils/Feature.php

```diff
@@ -70,11 +127,25 @@ public static function getFeature($name)
         return false;
     }
 
+    /**
+     * Gets all the features.
+     *
+     * @since 0.1.0
+     *
+     * @return array
+     */
     public static function getFeatures()
     {
         return self::$features;
     }
 
+    /**
+     * Sets the initial file of a feature.
```

Comment:
Some explanation about what the initial file is and what it is for would be helpful.